jcl/src/java.base/share/classes/java/lang/invoke/FieldGetterHandle.java

```diff
@@ -56,41 +61,41 @@ private final static MethodType fieldMethodType(Class<?> returnType, Class<?> ar
 	@FrameIteratorSkip
 	private final int    invokeExact_thunkArchetype_I(Object receiver, int argPlaceholder) {
 		if (Modifier.isVolatile(final_modifiers))
-			return getUnsafe().getIntVolatile(receiver, vmSlot+HEADER_SIZE);
+			return _unsafe.getIntVolatile(receiver, vmSlot+HEADER_SIZE);
 		else
-			return getUnsafe().getInt        (receiver, vmSlot+HEADER_SIZE);
+			return _unsafe.getInt        (receiver, vmSlot+HEADER_SIZE);
```

Comment:
Please add spaces around the `+` here and the other lines you've changed. I see they weren't there originally, but we might as well use the chance to fix it now.

Sorry, I should have realized how big the side-effects would be when changing the Unsafe getter.